lib/model.js

```diff
@@ -1630,7 +1630,7 @@ module.exports = (function() {
         }
       });
     } else {
-      options.fields = _.intersection(Object.keys(values), Object.keys(this.tableAttributes));
+      options.fields = Object.keys(this.tableAttributes);
```

Comment:
That's not exactly a proper fix, the old behaviour was very wrong aswell, updatedAt needs to be added to fields like we do in instance.save() i guess.